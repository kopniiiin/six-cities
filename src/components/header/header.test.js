import React from "react";
import {shallow} from "enzyme";

import Header from "./header";

describe(`snapshot test: Header component`, () => {
  it(`should render correctly`, () => expect(shallow(<Header/>)).toMatchSnapshot());

  it(`should render email`, () => expect(shallow(<Header email={`email`}/>)).toMatchSnapshot());

  it(`should render element`, () => expect(shallow(<Header><div/></Header>)).toMatchSnapshot());
});
