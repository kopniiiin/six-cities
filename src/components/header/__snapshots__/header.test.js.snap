// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: Header component should render correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <Link
          className="header__logo-link"
          to="/"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="img/logo.svg"
            width="81"
          />
        </Link>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <li
            className="header__nav-item user"
          >
            <Link
              className="header__nav-link header__nav-link--profile"
              to="/login"
            >
              <div
                className="header__avatar-wrapper user__avatar-wrapper"
              />
              <span
                className="header__login"
              >
                Sign in
              </span>
            </Link>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;

exports[`snapshot test: Header component should render email 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <Link
          className="header__logo-link"
          to="/"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="img/logo.svg"
            width="81"
          />
        </Link>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <li
            className="header__nav-item user"
          >
            <Link
              className="header__nav-link header__nav-link--profile"
              to="/favorites"
            >
              <div
                className="header__avatar-wrapper user__avatar-wrapper"
              />
              <span
                className="header__user-name user__name"
              >
                email
              </span>
            </Link>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
